### DISCUSSION TITLE
anyone want to help me come up with ideas for a project

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [Prodigy-Hacking/ProdigyMathGameHacking#886] [(A) Anything]
Anyone want to help me come up with ideas for a project? You can be a co-owner for whatever we decide to do, if you want like to tell me the idea you can @ me or email me here! my email is down below! :)

calebthehufflepuff@gmail.com

**Comment by SakuraKitsunee** [comment]
@BurnedWaffle do you want to help me make a project, if so do you have any cool ideas

**Reply by ArcerionDev** [reply]
feel free to open discussions like this but please refrain from pinging people if they haven't previously been in the conversation

**Reply by SakuraKitsunee** [reply]
oh ok will not do that anymore

**Reply by SakuraKitsunee** [reply]
should i delete the comment?

**Reply by ArcerionDev** [reply]
won't matter so dw about it

**Comment by ghost** [comment]
I think a good idea for an extension would be something kind of like a command runner except you have a hotkey (eg. Ctrl + Shift + Alt + V)
and it spams another hotkey (eg. Ctrl V) so it would repetitively paste whatever is in your clipboard (also a very good tool for spamming) @SakuraKitsunee

**Reply by SakuraKitsunee** [reply]
i could probably do that it might take awhile tho
